transcom/mymove
# Em mb 9676 update services counselor queue

## [Jira ticket](https://dp3.atlassian.net/browse/MB-9676) for this change

## Summary

This PR updates the services counselor queue to show moves that are within the office users GLBOC based on the moves origin duty location. This also updates the queue to have no differences for an office user if they are a marine. All marine moves will be displayed based only on their origin duty station GBLOC. 

## How to Verify Changes

To verify tests run `make server_test` and `make client_test`.

To verify locally through the UI complete the following steps:
- In two separate terminals run `make server_run` and `make client_run`
- In the customer UI create a two moves: one as not a marine service member, and one as a marine service member.  Ensure each origin duty location has the same GBLOC as the office user you intend to view the queue with. I believe the GLBLOC is "KKFA"  if you use the pre-created service counselor locally.
- Once you have created the moves, log into the office app and sign in as a services counselor. 

You should see both moves in the queue!

## Screenshots
![Screen Shot 2021-12-21 at 2 04 14 PM](https://user-images.githubusercontent.com/56279459/147008886-3a74f1d9-0865-4cb8-9a45-5bf71f59aab6.png)
